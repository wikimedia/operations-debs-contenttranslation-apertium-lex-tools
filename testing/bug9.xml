<rules>
  <rule>
    <match lemma="մարդ" tags="n.aa.*.*.*.px1sg"><select lemma="husband" tags="n.*"/></match>
  </rule>
</rules>
