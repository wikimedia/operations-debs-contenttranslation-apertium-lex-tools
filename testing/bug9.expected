^մարդ<n><aa><sg><nom><ind><px1sg>/husband<n><sg><nom><ind><px1sg>$ ^տուն<n><nn><sg><dat><def>/home<n><sg><dat><def>/house<n><sg><dat><def>/family<n><sg><dat><def>/stanza<n><sg><dat><def>/dynasty<n><sg><dat><def>/building<n><sg><dat><def>/apartment<n><sg><dat><def>/household<n><sg><dat><def>$ ^չ<adv>/not<adv>$ ^եմ<vaux><pres><p3><sg>/be<vbser><pres><p3><sg>$^:<sent>/:<sent>$
